src/qt/walletmodel.cpp

```diff
@@ -558,3 +558,8 @@ bool WalletModel::isMultiwallet()
 {
     return m_node.getWallets().size() > 1;
 }
+
+void WalletModel::AbortRescan() const
+{
+    wallet().AbortRescan();
```

Comment:
Nit, `m_wallet->AbortRescan()`?

Reply:
In commit "Add cancel button to rescan progress dialog"

Would suggest dropping this method and just calling model.wallet().abortRescan() directly. I don't think there's a benefit to wrapping individual `Wallet` calls inside a `WalletModel` methods unless they're doing extra work like synchronization or caching.

Reply:
Removed and called directly from walletview